### DISCUSSION TITLE
More stats

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#306] [Ideas]
Hi,

Maybe an idea to create more stats on the Monitoring page. Like:

- total number of file downloads
- total unique ip adresses
- total hits
- unique hits
- total bytes which have been downloaded
- total bytes downloaded that day
- number of unique users that day

Some examples and I am sure there are more.

Kind regards,
Henk - PH5HP

**Comment by rejetto** [comment]
hey, what are the most important to you?

**Reply by PH5HP** [reply]
Hi Rejetto,

Nothing is important, but it was nice to have some extra insight in the usage of the server. I found some of it in monitoring.

Another thing: I found out that every time I open Logs I have to doing the sort on time (last entry up) again. Could you make it so that it is rememberd and stored in the config?

Kind regards,
Henk - PH5HP

**Comment by PH5HP** [comment]
Hi,

Some ideas:

- stats beeing remembered in the admin console. Like total succesfull downloads, total data downloaded, unique users, total users, etc
- some tokens to use to display the stats in the layout of the server

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by PH5HP** [comment]
Hi Rejetto,

Still missing the stats like we had in version 2.xx of you server software. Also the total stats  were nice to have. It gave me an insight in usage during a time period. Also a way to display some of the stats in the layout would be nice. :-)

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Reply by W-i-n-7** [reply]
hey do you have discord? id like to talk with u a bit
: win_7

**Reply by PH5HP** [reply]
Hi,

Yes I have...

Name: pss0ft

Kind regards,
Henk - https://fileserver.ph5hp.nl
